tools/code-owners-parser/CodeOwnersParser/FileHelpers.cs

```diff
@@ -8,24 +8,22 @@ public static class FileHelpers
     {
         public static string GetFileContents(string fileOrUri)
         {
-            if (File.Exists(fileOrUri))
-            {
-                return File.ReadAllText(fileOrUri);
-            }
-
+            string fullPath = (new DirectoryInfo(fileOrUri)).FullName;
```

Comment:
Why are you creating a DirectoryInfo here?

Reply:
Changed as suggested. Use `Path.GetFullPath`